shared/Home/UseCases.tsx

```diff
@@ -106,45 +134,66 @@ export const engine = inngest.createFunction(
 
 const content = [
   {
-    title: "Durable workflows",
+    title: "LLM Chains",
     content: (
       <p>
-        Combine any complex series of tasks into a single reliable workflow.
-        Each task becomes a “step” which is automatically retried when errors
-        happen.
+        Get your LLM apps running in production without the complexity of glue
+        code and additional infrastructure.
       </p>
     ),
     bullets: [
-      <>Run steps in parallel or series or call a child workflow.</>,
+      <>Handle complex text-generation with chain-based post-processing.</>,
       <>
-        Add durable sleep to pause your workflow for days or weeks at a time.
+        Leverage Retrieval Augmented Generation (RAG) by querying vector stores
+        without complex interfaces and building ingestion functions.
```

Comment:
```suggestion
        without complex interfaces or building ingestion functions.
```

Reply:
Reworded this. I did want to highlight that you can use Inngest to build both components that power RAG (ingestion + querying).